packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/index.scss

```diff
@@ -92,7 +92,7 @@
     margin: 0 0 8px 0;
 
     &.new-plan-label {
-      background-image: url('../../../images/down-arrow.png');
+      // background-image: url('../../../images/down-arrow.png');
```

Comment:
Q: why remove this?

Reply:
This was prior to me addressing getting this image loading in Storybook, it needed to use `file-loader`.